.golangci.yaml

```diff
@@ -0,0 +1,101 @@
+run:
+  issues-exit-code: 1
+  timeout: 5m
+
+linters-settings:
+  lll:
+    line-length: 130
+  gocyclo:
+    min-complexity: 15
+  gci:
+    sections:
+      - standard
+      - default
+      - prefix(github.com/stacklok/minder)
```

Comment:
```suggestion
      - prefix(github.com/stacklok/trusty-sdk-go)
```

Reply:
OK, fixed the import path in the gci config and rebased, ready to go :+1: